engine/src/main/java/org/terasology/rendering/world/WorldRendererImpl.java

```diff
@@ -777,7 +739,8 @@ public Camera getActiveCamera() {
 
     @Override
     public Camera getLightCamera() {
-        return shadowMapCamera;
+        //FIXME: remove this method
```

Comment:
@Cervator: how do we know if a method is used in the API, the one that we shouldn't break because we fixed it for version 1?